recipe/run_test.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
+"""
+Test script for openjpeg.
+
+Currently main purpose is to check that the openjpeg library
+if linked dynamically to libpng, libtiff and libz as in 
+the latest build recipe.
+"""
+
+import subprocess
+
+# command we run to get output - for subprocess.check_output
+INSPECT_COMMAND = ['conda', 'inspect', 'linkages', 'openjpeg', '--name', '_test']
+
+# Currently we only check that this string exists in the output
+# of INSPECT_COMMAND.
+# This allows some difference in the endings (versions, extension etc)
+# but may need to be tightened up at some stage.
+# Need to be bytes to keep Python3 happy (subprocess.check_output returns bytes)
+REQ_LIBS = (b'lib/libtiff', b'lib/libpng', b'lib/libz')
```

Comment:
Don't think so as the comparison is `lib not in data`.
I guess we need something more elaborated to ensure they are not in the `system` field of the `conda-inspect` output.

However, because we are only building Linux-64 and there the library directory **is** `lib64` and not `lib` as in conda's prefix, this actually works.